docs/install/install-pro.md

```diff
@@ -8,109 +8,67 @@ If you already run Percona Server for MongoDB and wish to upgrade to Percona Ser
-            baseurl=http://repo.percona.com/private/[TOKENID]-[TOKEN]/psmdb-70-pro/yum/release/$releasever/RPMS/x86_64
-            enabled=1
-            gpgkey = https://repo.percona.com/yum/PERCONA-PACKAGING-KEY
-            ```    
-
-        2. Install Percona Server for MongoDB Pro packages    
-
-            ```{.bash .data-prompt="$"}
-            $ sudo yum install -y percona-server-mongodb-pro
+            ```{.bash data-prompt="$"}
+            $ sudo dpkg -i percona-release_latest.$(lsb_release -sc)_all.deb
             ```
-            
-        3. Start the server    
+
+        3. Update the local cache    
 
             ```{.bash .data-prompt="$"}
-            $ sudo systemctl start mongod
+            $ sudo apt update
             ```
 
-=== "ARM64"
+    === ":material-redhat: RHEL and derivatives"  
 
-    === ":material-debian: Debian and Ubuntu"    
+        Install `percona-release` using the following command:
 
-        1. Configure the repository. Create the `/etc/apt/sources.list.d/percona-psmdb-70-release.list` configuration file with the following contents:    
+        ```{.bash data-prompt="$"}
+        $ sudo yum install https://repo.percona.com/yum/percona-release-latest.noarch.rpm
+        ```
 
-            ```ini title="/etc/apt/sources.list.d/percona-psmdb-70-release.list"
-            deb http://repo.percona.com/private/[TOKENID]-[TOKEN]/psmdb-70-pro/apt/ [OPERATING_SYSTEM] main
-            ```    
+2. Enable the repository. Choose your preferable method:
 
-        2. Update the local cache    
+    === ":material-console: Command line"
 
-            ```{.bash .data-prompt="$"}
-            $ sudo apt update
-            ```    
+        Run the following command and pass your credentials to the Pro repository:
 
-        3. Install Percona Server for MongoDB Pro packages    
+        ```{.bash .data-prompt="$"}
+        $ sudo percona-release enable psmdb-70-pro release --user_name <Your Customer ID> --repo_token  <Your PRO repository token>
+        ```
 
-            ```{.bash .data-prompt="$"}
-            $ sudo apt install -y percona-server-mongodb-pro
-            ```    
+    === ":octicons-file-code-24: Configuration file"
 
-        4. Start the server    
+        Create the `/root/.percona-private-repos.config` configuration file with the following content:
 
-            ```{.bash .data-prompt="$"}
-            $ sudo systemctl start mongod
-            ```    
+        ```ini title="/root/.percona-private-repos.config"
+        [psmdb-70-pro]
+        USER_NAME=<Your Customer ID>
+        REPO_TOKEN=<Your PRO repository token>
+        ```    
 
-    === ":material-redhat: RHEL and derivatives"    
+3. Install Percona Server for MongoDB Pro packages    
 
-        1. Configure the repository. Create the `/etc/yum.repos.d/psmdb-pro.repo` configuration file with the following contents:    
+    ```{.bash .data-prompt="$"}
+    $ sudo apt install -y percona-server-mongodb-pro
```

Comment:
There is no command for RHEL and derivatives

Reply:
Fixed